### ISSUE TITLE
Add OAuth with Google and Github

### ISSUE THREAD

**Issue by AbhayAysola** [ppalone/crypt#15]
Currently, the method of signing up is to enter name, email and password. Instead of that, a sign in with Github or Google option would be simpler and easier to sign up and log in. 
[Google Sign in Docs](https://developers.google.com/identity/sign-in/web/sign-in)

**Comment by ppalone** [comment]
I was will be a great enhancement. As there will be no need of email verification and no need to remember password. One click and you are in the app 😊

**Comment by AbhayAysola** [comment]
Exactly, I will try to integrate Google later, and I'll let you know how it goes :)

**Comment by ppalone** [comment]
`passport-google-oauth2` allows you to integrate Google OAuth easily.

http://www.passportjs.org/packages/passport-google-oauth2/

**Comment by AbhayAysola** [comment]
All right, thanks man. L
